README.markdown

```diff
@@ -7795,10 +7797,46 @@ tcpsock:setoption
 
 This function is added for [LuaSocket](http://w3.impa.br/~diego/software/luasocket/tcp.html) API compatibility and does nothing for now. Its functionality will be implemented in future.
```

Comment:
Its functionality is implemented v0.10.18.

Reply:
ok